extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -424,10 +426,14 @@ public function saveUserData() {
 			 */
 			$userIdentityBox = new UserIdentityBox( $user );
 
-			if ( !empty( $userData->website ) && 0 !== strpos( $userData->website, 'http' ) ) {
+			if ( !empty( $userData->website ) && strpos( $userData->website, 'http' ) !== 0 ) {
 				$userData->website = 'http://' . $userData->website;
 			}
 
+			if ( !empty( $userData->fbPage ) && strpos( $userData->website, self::FBPAGE_PREFIX ) !== 0 ) {
```

Comment:
UI looks like this:
![screen](https://cloud.githubusercontent.com/assets/2725894/6267221/5fff9f7a-b842-11e4-9453-cfebb8a7add4.png)

```facebook.com``` is already prepopulated - in the same way as ```@``` for twitter is. Do you think we need to add separate hint?